server.go

```diff
@@ -3,16 +3,35 @@ package qsse
 import (
 	"crypto/tls"
 	"net/http"
+	"time"
 
 	"github.com/go-errors/errors"
 	"github.com/lucas-clemente/quic-go"
 	"github.com/snapp-incubator/qsse/auth"
 	"github.com/snapp-incubator/qsse/internal"
 )
 
+var DefCleaningInterval = 10 * time.Second //nolint:gochecknoglobals
```

Comment:
@mehditeymorian Jan, I think we can define it as const value. Do you have any other consideration about it?